dsl/pace/dsl/stencil.py

```diff
@@ -111,6 +114,11 @@ def stencil_kwargs(self):
         }
         if not self.is_gpu_backend:
             kwargs.pop("device_sync", None)
+        # Note: this assure the backward compatibility between v36 and v37
+        if "skip_passes" in kwargs:
+            kwargs["oir_pipeline"] = StencilConfig._get_oir_pipeline(
+                kwargs.pop("skip_passes")
+            )
```

Comment:
This argument isn't needed when skip_passes is False, correct? I ask because in practice we currently always use skip_passes (as I understand it), so the tests wouldn't catch this.

Reply:
This is a part of GT4Py where I don't know the details, but I believe so. This is giving me grief in the utest anyway, I'll have to revisit.